### PR TITLE
Adding Error Modal to Farming Page

### PR DESCRIPTION
**Pull request by saltoriousSIG**

### CODE REVIEW

File: src/components/Errors/UserErrorScreen.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import error from '../../assets/error.svg'
+import styled from 'styled-components'
+
+const UserErrorScreen = props => {
+  const [visible, setVisible] = useState()
+
+  useEffect(() => {
+    setVisible(props.isVisible)
+  }, [props.isVisible])
+  console.log(visible)
```
Comment (famole) [inline src/components/Errors/UserErrorScreen.js]: let's remove this console.log

File: src/components/Farming/Deposit/index.js [hunk 2]
```diff
@@ -36,18 +36,26 @@ const Deposit = props => {
   const handleDeposit = () => {
     deposit()
       .then(x => {
-        if (x) {
+        console.log(typeof x)
```
Comment (famole) [inline src/components/Farming/Deposit/index.js]: can we remove this console.log?

File: src/components/Farming/Deposit/index.js [hunk 3]
```diff
@@ -36,18 +36,26 @@ const Deposit = props => {
...
+        if (x && x.message === undefined) {
+          console.log(x, typeof x)
```
Comment (famole) [inline src/components/Farming/Deposit/index.js]: same here

File: src/components/Farming/DepositModal/index.js [hunk 4]
```diff
@@ -50,112 +53,129 @@ const DepositModal = props => {
   const handleTransactionComplete = () => {
     props.handleModalClose()
   }
+  const handleError = err => {
+    console.log(err)
```
Comment (famole) [inline src/components/Farming/DepositModal/index.js]: and this one :)

File: src/components/Farming/DepositTable/index.js [hunk 5]
```diff
@@ -63,6 +68,17 @@ const DepositTable = props => {
     ],
   })
   const results = fuse.search(props.searchValue)
+
+  const handleError = err => {
+    console.log(err)
```
Comment (famole) [inline src/components/Farming/DepositTable/index.js]: console.log

**Review by famole — CHANGES_REQUESTED** [review]
Nice work @saltoriousSIG, left a few comments

**Comment by saltoriousSIG** [comment]
I'm the worst with leaving console.logs! Just removed them all.

**Pull request merged**